app/javascript/turbo.js

```diff
@@ -12,17 +11,11 @@ addEventListener('turbo:before-fetch-request', event => {
   const { fetchOptions } = event.detail
 
   // command invoked and busy
-  if (meta.busy) {
+  if (self.TurboBoost?.Commands?.busy) {
     let acceptHeaders = ['text/vnd.turbo-boost.html', fetchOptions.headers['Accept']]
     acceptHeaders = acceptHeaders.filter(entry => entry && entry.trim().length > 0).join(', ')
     fetchOptions.headers['Accept'] = acceptHeaders
-    fetchOptions.headers['TurboBoost-Token'] = meta.token
   }
-
-  // always send state
-  state.payloadChunks.forEach((chunk, i) => {
-    fetchOptions.headers[`TurboBoost-State-${i.toString().padStart(4, '0')}`] = chunk
-  })
```

Comment:
Previously we always sent state. We may want to consider bringing this back when we put more attention on state management and optimistic updates.